src/pages/WebMeetWaitingRoom.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+import "../App.css";
+import "./home.css";
+import * as patientService from "../services/patient";
+import { getDateString, getTimeString, getToday } from "../utils";
+import { BOOKING_STATUS } from "../constants/constants";
+
+const WebMeetWaitingRoom = ({ patientDetails }) => {
+  const [appointment, setAppointment] = useState({});
+  const today = getToday();
+  useEffect(() => {
+    window.scrollTo(0, 0);
+    getAppointmentDetail();
+  }, []);
+
+  const getAppointmentDetail = async () => {
+    try {
+      const response = await patientService.getAppointmentDetailsByPatientId(
+        patientDetails.patientId
+      );
+      setAppointment(response.appointment);
+
+      response.appointment.eventStatusDesc === BOOKING_STATUS.confirmed &&
+        (await patientService.UpdateAppointmentStatus(
+          response.appointment.organizationEventBookingId,
+          patientDetails.patientId
+        ));
+
+    } catch (err) {}
+  };
+
+  console.log(appointment);
+
+  return (
+    <>
+      {getDateString(appointment.eventStartTime) === today &&
```

Comment:
better to put `{getDateString(appointment.eventStartTime)` inside `useMemo` so it doesn't run every re-render

Reply:
resolved